src/scrapers/twitter.iced

```diff
@@ -280,15 +280,19 @@ exports.TwitterScraper = class TwitterScraper extends BaseScraper
     inside = ws_normalize inside
     proof_text_check = ws_normalize proof_text_check
 
-    @log "+ Checking tweet '#{inside}' for signature '#{proof_text_check}'"
+    rxx_text = proof_text_check.replace(" on Keybase.io.", " on (Keybase.io|https://t\\.co/\\S*|http://Keybase\\.io\\s)\\.")
```

Comment:
I just took a look at Taco von Taco's latest proof in my own browser, and it looks like the "space thing" at the end that they're printing is the U+00A0 nonbreaking space. Go figure.

Reply:
aha that explains it!